packages/spiderfier/services/managers/spider-manager.ts

```diff
@@ -0,0 +1,175 @@
+import {Injectable, NgZone, SkipSelf} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+import {OverlappingMarkerSpiderfier} from 'ts-overlapping-marker-spiderfier';
+
+import {MarkerManager} from '../../../core/services/managers/marker-manager';
```

Comment:
from `'@agm/core'` and for the following lines also

Reply:
Updated